v3/plugins/rogalmic/vscode-bash-debug/0.3.9/meta.yaml

```
@@ -0,0 +1,15 @@
+publisher: rogalmic
+apiVersion: v2
+name: bash-debug
+version: 0.3.9
+type: VS Code extension
+displayName: Bash Debug
+title: A bash debugger GUI frontend based on awesome bashdb scripts (bashdb now included in package).
+description: This is a SIMPLE bashdb debugger frontend. Useful for learning bash shell usage and writing simple scripts.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/rogalmic/vscode-bash-debug
+category: Other
+firstPublicationDate: '2020-07-4'
```

maybe use the 2 digits (2020-07-04)
```suggestion
firstPublicationDate: '2020-07-04'
```